restcomm.android.sdk/src/main/java/org/restcomm/android/sdk/RCDevice.java

```diff
@@ -313,7 +313,7 @@
    public static String EXTRA_VIDEO_ENABLED = "org.restcomm.android.sdk.VIDEO_ENABLED";
 
    // Notification ids for calls and messages. Key is the sender and value in the notification id. We mainly need those
-   // to make sure that notifications from a specific user are shown in a single slot, to avoid confusing the user. This means
+   // to make sure that notifications from a specific user are shown onReconfiguredin a single slot, to avoid confusing the user. This means
```

Comment:
Can you elaborate on onReconfiguredin? Is it a typo?

Reply:
typo